### ISSUE TITLE
Add support for styles as a map

### ISSUE THREAD

**Issue by kennyjwilli** [jeluard/hipo#26]
Is would be incredibly helpful to be able to add styles to an element via a map with the below syntax:
```
(hipo.core/create [:span {:style {:background-color "blue"}} "a span"])
```

**Comment by jeluard** [comment]
I'd rather have this done using an `interceptor`. Let me think about it more.

**Comment by jeluard** [comment]
I pushed a relevant change in `master`, you can now do something like:

```clojure
(hipo/create [:span {:style {:background-color "blue"}]
             {:attribute-handlers [hipo.attribute/style-handler]})
```

I prefer to stick default behavior to the original `hiccup` syntax.

Let me know what you think.

**Comment by kennyjwilli** [comment]
Looks good. Perhaps `(aset %1 "style" (name k) (name v))` so that you can do something like
```
[:span {:style {:background-color :blue}]
```

**Comment by jeluard** [comment]
Besides color and a couple other cases keywords are probably not that useful so I'd rather have a simple rule: strings everywhere.
if you want to tweak this handler you can provide your own anyway!

**Comment by jeluard** [comment]
Released as version `0.5.1`.
